packages/react-dev-utils/getProcessForPort.js

```diff
@@ -43,7 +43,7 @@ function getProcessCommand(processId, processDirectory) {
 }
 
 function getDirectoryOfProcessById(processId) {
-  return execSync('lsof -p '+ processId + ' | grep cwd | awk \'{print $9}\'', execOptions).trim();
+  return execSync('lsof -p '+ processId + ' | awk '$4=="cwd"' | awk \'{print $9}\'', execOptions).trim();
```

Comment:
I'm not an awk expert, but looks like we could also combine the two `awk` commands: instead of piping to a separate command, we would just do `awk '$4 == "cwd" { print $9 }'`.